runtime/onert/api/python/package/experimental/train/losses/cce.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+from .loss import LossFunction
+
+
+class CategoricalCrossentropy(LossFunction):
+    """
+    Categorical Cross-Entropy Loss Function with reduction type.
+    """
+    def __init__(self, reduction="mean"):
+        """
+        Initialize the Categorical Cross-Entropy loss function.
+        Args:
+            reduction (str): Reduction type ('mean', 'sum').
+        """
+        super().__init__(reduction)
+
+    def __call__(self, y_true, y_pred):
+        """
+        Compute the Categorical Cross-Entropy loss.
+        Args:
+            y_true (np.ndarray): One-hot encoded ground truth values.
+            y_pred (np.ndarray): Predicted probabilities.
+        Returns:
+            float or np.ndarray: Computed loss value(s).
+        """
+        epsilon = 1e-7  # Prevent log(0)
+        y_pred = np.clip(y_pred, epsilon, 1 - epsilon)
+        loss = -np.sum(y_true * np.log(y_pred), axis=1)
+
+        if self.reduction == "mean":
+            return np.mean(loss)
+        elif self.reduction == "sum":
+            return np.sum(loss)
```

Comment:
May I ask why loss function is implemented here? 
I thought the loss function executed inside the ONERT here -  https://github.com/Samsung/ONE/blob/da3fdd292d288f22d3abd393845c74c18381ea76/runtime/onert/backend/train/ops/LossCategoricalCrossentropyLayer.h#L41-L42

Reply:
Right, the loss function run inside there. It seems that this function is unnecessary.

Reply:
I removed it.